LibreNMS/Util/GitHub.php

```diff
@@ -391,14 +389,14 @@ public function createChangelog($write = true)
             $this->getPullRequest();
         }
 
-        if (! isset($previous_release['published_at'])) {
+        if (! isset($previous_release['created_at'])) {
```

Comment:
@murrant I think this was a bug.

>The created_at attribute is the date of the commit used for the release, and not the date when the release was drafted or published.

We want to compare from the tag date rather than the "github release" date